unit_07/hypothesis_test_practice_activity.Rmd

```diff
@@ -27,51 +44,161 @@ salary_potential   <- read_csv(file = paste0(base_url, 'salary_potential.csv'))
-                               ## and exposes the variable, outside a data.frame
-  t.test(in_state_tuition, mu = public_school_cost_1985)
-
-?%$%
+  filter(type == 'Public') %>% 
+  select(in_state_tuition) %$%
+  t.test(
+    x=in_state_tuition, 
+    mu=public_school_cost_1985
+    )
 ```
 
+1. Are the tuition costs different? 
+2. Is this a big difference or a little difference? What makes you think this? 
+
 # Two-Sample t-test 
 
+Do schools whose students are more diverse charge higher or lower tuition? For the purposes of this test, define a diverse school to be a school where more than 40% of its student body identify as members of historically under-represented groups. 
+
+[UC Berkeley defines](https://diversity.berkeley.edu/reports-data/diversity-data-dashboard) under-represented groups to be groups who identify as: 
+
+- Black or African-American 
+- Chicano or Latino 
+- Native American / Alaska Native 
+- Pacific Islander
+
+```{r create diverse school indicator}
+diverse_school <- diversity_school %>% 
+  mutate(urm = case_when(
+    category == 'American Indian / Alaska Native' ~ 1, 
+    category == 'Black' ~ 1, 
+    category == 'Hispanic' ~ 1, 
+    category == 'Native Hawaiian / Pacific Islander' ~ 1, 
+    TRUE ~ 0)) %>% 
+  filter(urm == 1) %>%
+  group_by(name) %>% 
+  summarise(
+    urm_percentage = sum(enrollment) / mean(total_enrollment)) %>% 
+  mutate(diverse_school = urm_percentage >= 0.40)
+```
+
+```{r plot school urm distribution}
+diverse_school %>% 
+  filter(urm_percentage < 1) %>% 
+  ggplot() + 
+  aes(x = urm_percentage) + 
+  geom_histogram()
+```
+
+```{r}
+tuition_cost %>% 
+  left_join(diverse_school) %$%
+  t.test(in_state_total ~ diverse_school)
+```
+
 # Paired t-test 
 
+Do in-state, or out-of-state students pay more for tuition? To begin, create a scatter plot using the `tuition_cost` data. 
+
+- On the x-axis plot the `in_state_total` tuition
+- On the y-axis plot the `out_of_state_total` tuition
+- Color the points of the plot by the `type` of school. (If it helps, you may filter out the "other" types of schools). 
+- If this plot would be enhanced by showing points on different sets of axes, you might consider using the `fact_grid()` layer to your plot, and set either the `rows` or the `cols` argument of this layer to take in the variable `type` using the call `facet_grid(cols = vars(type))`. 
+
+```{r}
+tuition_cost %>% 
+  filter(type %in% c('For Profit', 'Private', 'Public')) %>%
+  ggplot() +  
+  aes(x=in_state_total, y=out_of_state_total, color=type) + 
+  geom_point(alpha = 0.3) + 
+  facet_grid(cols = vars(type))
+```
+Given what you have seen, how would you recommend proceeding with your test? Proceed in the way that you think is most reasonable. State the assumptions for your test, evaluate whether they are satisfied, and conduct the test, describing what you have learned about the statistics, and also what the practical meaning of these statistics are. 
```

Comment:
the learning is about the effect or the model parameters or the difference in tuition, not the statistics, right?